src/Router.php

```diff
@@ -239,14 +240,18 @@ public static function matchRequestedUrl()
      *
      * Router::redirectToRoute("home.index"); // redirects to "/"
      */
-    public static function redirectToRoute(string $name, int $status = Redirection::SEE_OTHER): void
+    public static function redirectToRoute(string $name, array $routeParams = [], int $status = Redirection::SEE_OTHER): void
     {
         if (!self::hasRoute($name)) {
             throw (new RouteNotFoundException("route $name not found"))->setRoute($name);
         }
 
         $url = self::getRouteUrl($name);
 
+        if (!empty($routeParams)) {
```

Comment:
@hstrychalski could you please try to use the second parameter of `self::hasRoute()` to add the `$routeParams` instead of using the method `self::appendUrlParams()`? When you will use the second parameters, you will beneficiate of a whole logic for filling the route named parameters `/user/{user}/post/{post}` for example with `["user" => 42, "post" => 13]`

Thanks!